### PR TITLE
Create sk8944.md

### PR DESCRIPTION
**Pull request by siddharthkandpal**
Tabulation: Week 1

### CODE REVIEW

**Comment by nickelpro** [comment]
So this needs to create at least some sort of entry in your design notebook.

Even if you didn't get through anything this week, make note of that, and any problems you encountered or even if you didn't have the time/resources to complete an onboarding lab.

If you need any help populating the entry, please reach out here or in the Discord. You can add additional commits to your fork in and they'll appear in this PR. Once an entry has been made I'll merge this

**Comment by siddharthkandpal** [comment]
Hi Vito,
Yeah, I figured that out. Should’ve done the lab first and then populated
the entry.

Will keep that in mind. Yes, I’m onto it. Thanks for the prompt revert.


Siddharth.

On Sat, Jan 28, 2023 at 6:43 PM Vito Gamberini ***@***.***>
wrote:

> So this needs to create at least some sort of entry in your design
> notebook.
>
> Even if you didn't get through anything this week, make note of that, and
> any problems you encountered or even if you didn't have the time/resources
> to complete an onboarding lab.
>
> If you need any help populating the entry, please reach out here or in the
> Discord. You can add additional commits to your fork in and they'll appear
> in this PR. Once an entry has been made I'll merge this
>
> —
> Reply to this email directly, view it on GitHub
> <https://urldefense.proofpoint.com/v2/url?u=https-3A__github.com_NYU-2DProcessor-2DDesign_nyu-2Dprocessor-2Ddesign.github.io_pull_17-23issuecomment-2D1407512955&d=DwMFaQ&c=slrrB7dE8n7gBJbeO0g-IQ&r=atLPJQCZj-SMMI3fbQqtBw&m=aibtTDsN_H6xQkShTQFzHjdPORS-wonGHsw_vJTLhfS9zHDHogu_dCHc_EGv3ZkU&s=Htr0plYModMcS5ox4T3-74VWf7slUDxyMXhuj5En5l8&e=>,
> or unsubscribe
> <https://urldefense.proofpoint.com/v2/url?u=https-3A__github.com_notifications_unsubscribe-2Dauth_A5NVLFYEOUEKB5OTMAFNS6TWUWVKBANCNFSM6AAAAAAUJ36P4A&d=DwMFaQ&c=slrrB7dE8n7gBJbeO0g-IQ&r=atLPJQCZj-SMMI3fbQqtBw&m=aibtTDsN_H6xQkShTQFzHjdPORS-wonGHsw_vJTLhfS9zHDHogu_dCHc_EGv3ZkU&s=JZvncl3LlRzAlgtRQ1JNtuvaj6BUVVHpZD53i0YCryA&e=>
> .
> You are receiving this because you authored the thread.Message ID:
> <NYU-Processor-Design/nyu-processor-design.github.io/pull/17/c1407512955@
> github.com>
>
-- 
Siddharth Kandpal, NYU EE 23.

File: src/design_notebooks/2023spring/sk8944.md [hunk 1]
```diff
@@ -0,0 +1 @@
+
```
Comment (siddharthkandpal) [inline src/design_notebooks/2023spring/sk8944.md:1]: Week 1:

Onboarding went on pretty smoothly. Got a taste of the environment we'll be using to design our processor.

Attempted and finished Lab 1. Got exposed to CMake Toolchain, and how CML works.
While attempting the quests for Lab 1:
*make* des not work on windows. Would not geenrate a makefile and thus used cmake --build . instead.
When attempted to build on VsCode, ran into some minor issues with the header file missing directory even when specified in the CML using target_env_variables.

Also, Started reading up on using verilator.

**Comment by nickelpro** [comment]
Merging this, for next week pay attention to the format given in [the Design Notebook guide](https://nyu-processor-design.github.io/getting_started/notebooks/01_first_design.html). If you need a guide on how the markdown format works, take a look at [the cheatsheet](https://commonmark.org/help/) that's linked in the guide.

**Comment by siddharthkandpal** [comment]
Will follow the format henceforth, totally forgot about the modus operandi.
Apologies.


On Sun, Jan 29, 2023 at 2:06 PM Vito Gamberini ***@***.***>
wrote:

> Merged #17
> <https://urldefense.proofpoint.com/v2/url?u=https-3A__github.com_NYU-2DProcessor-2DDesign_nyu-2Dprocessor-2Ddesign.github.io_pull_17&d=DwMFaQ&c=slrrB7dE8n7gBJbeO0g-IQ&r=atLPJQCZj-SMMI3fbQqtBw&m=iF2OKZ-XdCy_liuWcN98aErnmL46_vgpdtgh53pR9V3B93H_sCMsOdgbioz05qcR&s=R7OUFyDFeIMl0qI6T9CEK2gUIENGMUapmDmFSN0XN2k&e=>
> into main.
>
> —
> Reply to this email directly, view it on GitHub
> <https://urldefense.proofpoint.com/v2/url?u=https-3A__github.com_NYU-2DProcessor-2DDesign_nyu-2Dprocessor-2Ddesign.github.io_pull_17-23event-2D8382451841&d=DwMFaQ&c=slrrB7dE8n7gBJbeO0g-IQ&r=atLPJQCZj-SMMI3fbQqtBw&m=iF2OKZ-XdCy_liuWcN98aErnmL46_vgpdtgh53pR9V3B93H_sCMsOdgbioz05qcR&s=ykWcCDZH8P3EbeJBCUsd44O7TqB_s3fEVQQ3Is-GxCk&e=>,
> or unsubscribe
> <https://urldefense.proofpoint.com/v2/url?u=https-3A__github.com_notifications_unsubscribe-2Dauth_A5NVLF2G2MEQAVNHZXAUXMLWU25THANCNFSM6AAAAAAUJ36P4A&d=DwMFaQ&c=slrrB7dE8n7gBJbeO0g-IQ&r=atLPJQCZj-SMMI3fbQqtBw&m=iF2OKZ-XdCy_liuWcN98aErnmL46_vgpdtgh53pR9V3B93H_sCMsOdgbioz05qcR&s=diu5Lg6O_hjDNnErqvKlACfWcS6lkanMglggyY40H_c&e=>
> .
> You are receiving this because you authored the thread.Message ID:
> <NYU-Processor-Design/nyu-processor-design.github.
> ***@***.***>
>
-- 
Siddharth Kandpal, NYU EE 23.

**Pull request merged**